app/modules/conversations/conversation/conversation_service.py

```
@@ -936,6 +936,33 @@ async def _generate_and_stream_ai_response(
+                    len(attachment_ids),
+                    attachment_ids,
+                )
+            text_attachments = await self._prepare_text_attachments(
+                attachment_ids or []
+            )
+
+            # Build additional_context from text attachments
+            additional_context = ""
+            if text_attachments:
+                text_parts = []
+                for att_id, att_data in text_attachments.items():
```

Ruff (0.14.14)

[warning] Loop control variable `att_id` not used within loop body

Rename unused `att_id` to `_att_id`

(B007)